docs/book/response.md

```diff
@@ -138,3 +138,16 @@ $response->setContent(<<<EOS
 </html>
 EOS);
 ```
+
+### handle Headers already sent
+
+> Available since version 2.13.0, we can handle header already sent by pass callable via `Response::setHeadersSentHandler()`:
```

Comment:
```suggestion
> Available since version 2.13.0

We can handle header already sent by pass callable via `Response::setHeadersSentHandler()`:
```

The goal is to provide a consistent reference for the reader. See also:

* https://docs.laminas.dev/laminas-hydrator/v3/strategies/hydrator/
* https://docs.laminas.dev/laminas-inputfilter/optional-input-filters/

Reply:
thank you, applied.